### ISSUE TITLE
Lid/Top cover

### ISSUE THREAD

**Issue by geotibi** [pianoforte33/food-dispenser#1]
Very nice project. I guess I will try something similar using ESPHome with Home Assistant. Do you have the 3D model of the Lid/Top cover part?

**Comment by pianoforte33** [comment]
Yes, you can find the 3D models on [Printables](https://www.printables.com/model/1045926-food-dispenser). While you are screwing in the hinges, be careful, as the piece on the body of the dispenser may break. I suggest using smaller screws or waiting until I fix the issue.
EDIT: now the 3d model are fixed
